internal/internal.go

```diff
@@ -237,6 +240,15 @@ func (p *envoyExtAuthzGrpcServer) listen() {
 		addr = "grpc://" + addr
 	}
 
+	if p.cfg.EnablePrometheusMetrics {
+		summaryAuthzDuration := prometheus.NewSummary(prometheus.SummaryOpts{
+			Name: "grpc_authz_request_duration_ms",
```

Comment:
It seems like this would be the gRPC equivalent of `http_request_duration_seconds`? I guess it's unfortunate that the HTTP metrics is _"seconds"_, so "ms" might be a better choice... :+1:

Reply:
Sure. Most grpc calls have duration about 2-10ms.

Reply:
`grpc_authz_request_duration_ms` sgtm